package.json

```diff
@@ -2644,6 +2644,16 @@
 				},
 				"description": "The color used for indicating that a pull request is merged."
 			},
+			{
+				"id": "pullRequests.merged.foreground",
+				"defaults": {
+					"dark": "#FFD9FE",
+					"light": "#FFD9FE",
+					"highContrast": "editor.foreground",
+					"highContrastLight": "editor.foreground"
```

Comment:
I'm happy for these to be customizable, but in general it's good to make the colors derive from an existing color. In this case, I think all 4 options here (and for the other new colors) should derive from the current color: `button.foreground`. This will create the least amount of disruption for users, and give an opportunity to for the color to be automatically configured to something that makes sense with their theme.

Reply:
If it's better in your theme, I would also be fine with `button.secondaryForeground`.

Reply:
Thanks for the review, and great point! I'll make them as such!

Reply:
I was thinking, and what about using the original GitHub colors to look exactly like the website? Background colors refer to them, so why not the foregrounds?

Reply:
If you're referring to the open, merged, and draft background colors then matching the website is fine if we don't already do it. I would have to double check and make sure that the colors don't look odd in the default VS Code dark and light themes.

For the foreground, I think it's just black or white on github.com, isn't it?

Reply:
Yeah and I thought we could default to white/black instead of button foreground since the background color is fixed, it would be dangerous to let the foreground be what the theme says (it will break everywhere)

Reply:
Can we please have the option to customize all of them (because it would make at least my theme much more pretty) and have all of them default to this color?

Reply:
Sure, that's fine for me. Let's have `pullRequests.statusOpenForeground`, then have the other `pullRequests.status*Foreground` values default to `pullRequests.statusOpenForeground`.

Reply:
Sure thing, I'll apply it now